CompetitionTemplate/src/competition/subsystems/drive/PoseSubsystem.java

```diff
@@ -29,6 +30,12 @@
     public DistanceSensor leftDistanceSensor;
     public DistanceSensor rightDistanceSensor;
     
+    public XEncoder leftDriveEncoder;
```

Comment:
oops, yeah. I forgot to delete the distance sensors before adding the encoders...right?